base/file.jl

```diff
@@ -280,6 +281,18 @@ function rm(path::AbstractString; force::Bool=false, recursive::Bool=false)
     end
 end
 
+"""
+    rmrf(path::AbstractString)
+
+Forcibly and recursively remove the provided path.
+Shorthand for `rm(path, force=true, recursive=true)`.
+
+See also [`rm`](@ref).
+
+!!! compat "Julia 1.3"
+    This function requires Julia 1.3 or later.
+"""
+rmrf(path::AbstractString) = rm(path, force=true, recursive=true)
```

Comment:
```suggestion
rmrf(path) = rm(path, force=true, recursive=true)
```

Reply:
Agree. No reason to constrain this.

Reply:
Just did it originally for consistency with `rm`.